scripts/integration-tests.sh

```diff
@@ -14,6 +14,8 @@ cabal test llvm-integration
 
 # The runDirectoryTest.sh script expects the following env vars to be set
 export PLUGIN_DIR=$(nix build github:runtimeverification/blockchain-k-plugin/$PLUGIN_VERSION --no-link --json | jq -r '.[].outputs | to_entries[].value')
+export SECP256K1_DIR=$(nix build nixpkgs#secp256k1 --no-link --json | jq -r '.[].outputs | to_entries[].value')
```

Comment:
This seems to be necessary now... root cause is that we don't actually get a linkable plugin library from including the `blockchain-c-plugin`, we rather get the sources and have to build things ourselves.
Also see the [change to the plugin build and derivation](https://github.com/runtimeverification/blockchain-k-plugin/pull/172/files) for context.